indico/migrations/versions/20231222_1408_b37cbc4bb129_link_events_to_reservation_occurrences.py

```diff
@@ -0,0 +1,72 @@
+"""Link events to reservation occurrences.
+
+Revision ID: b37cbc4bb129
+Revises: e2b69fe5155d
+Create Date: 2023-12-22 14:08:45.669303
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from enum import Enum
+from indico.core.db.sqlalchemy import PyIntEnum
+
+
+# revision identifiers, used by Alembic.
+revision = 'b37cbc4bb129'
+down_revision = 'e2b69fe5155d'
+branch_labels = None
+depends_on = None
+
+
+class _LinkType(int, Enum):
+    category = 1
+    event = 2
+    contribution = 3
+    subcontribution = 4
+    session = 5
+    session_block = 6
+
+
+excluded_link_types = {_LinkType.category, _LinkType.subcontribution, _LinkType.session}
+
+def upgrade():
+    op.create_table('reservation_occurrence_links',
```

Comment:
Don't you want to rename/update the existing table?

Reply:
I'd rather remove the existing table.